int32 start_index
int32 end_index
planner/PoseJ[] joints_pose_array_1
planner/PoseJ[] joints_pose_array_2
